vignettes/pr_review_guidance.Rmd

```diff
@@ -88,6 +91,69 @@ knitr::include_graphics("./pr_review_actions.png")
 
 Please don't hesitate to reach out to the `{admiral}` team on [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) or through the [GitHub Issues](https://github.com/pharmaverse/admiral/issues) tracker if you think this checklist needs to be amended or further clarity is needed on a check box item. 
 
+## GitHub Actions/Workflows
+
+The `task-list-completed` workflow is one of the several workflows/actions used within `{admiral}`. These workflows live in the `.github/workflows` folder and it is important to understand their use and how to remedy if the workflow fails. Workflows defined here are responsible for assuring high package quality standards without compromising performance, security, or reproducibility. 
```

Comment:
Reminder later to add here where the source workflows are housed, e.g. {admiraldev} or {admiralci} once we decide